src/commands/configuration/FeedbackChannel.ts

```diff
@@ -0,0 +1,206 @@
+                    });
+
+                    console.log(error);
+
+                    // client.log(
+                    //     error,
+                    //     `${this.data.name} Command - Add Feedback Channel`,
+                    //     `**Channel:** <#${interaction.channel?.id}>
+                    // 		**User:** <@${interaction.user.id}>
+                    // 		**Guild:** ${interaction.guild.name} (${interaction.guildId})\n`
+                    // );
+                }
+                break;
+            }
+            case "remove": {
+
+                const feedbackTeams = await FeedbackChannels.find({
+                    guildId: interaction.guildId
+                });
+
+                if (feedbackTeams.length == 0) {
+                    await interaction.reply({
+                        content: "There are no feedback channels to be removed",
+                        ephemeral: true
+                    })
+                    return;
+                }
+
+                const customId = uuidv4();
+
+                const teamSelect = new Select(
+                    "team",
+                    "Select a team to stop sending feedback to",
+                    feedbackTeams.map(({ label, channelId, id }) => ({
+                        label: `${label} | #${interaction.guild.channels.cache.get(channelId)?.name}`,
+                        value: id
+                    })),
+                    1,
+                    `${customId}_0`
+                );
+
+                const selectInteraction = await interaction.reply({
+                    content: "Select a team to remove from feedback",
+                    components: [
+                        new ActionRowBuilder<Select>().addComponents(teamSelect),
+                        new Buttons(customId) as ActionRowBuilder<ButtonBuilder>
+                    ],
+                    fetchReply: true,
+                    ephemeral: true
+                });
+
+                const response = await teamSelect.waitForResponse(
+                    `${customId}_0`,
+                    selectInteraction,
+                    interaction,
+                    true
+                );
+
+                if (!response || response === "Timed out" || !response[0]) {
+                    await interaction.followUp({
+                        content: "An error occurred",
+                        ephemeral: false
+                    })
+                    return;
+                }
+
+                const team = await FeedbackChannels.findById(response[0]);
+
+                if (!team) {
+                    await interaction.followUp({
+                        content: "Team not found",
+                        ephemeral: false
+                    })
+                    return;
+                }
+
+                try {
+                    await team.deleteOne();
+                    await interaction.editReply({
```

Comment:
No need for `await`.

Reply:
Only for `await interaction.editReply` right? since if not then ig it'll send the success reply before the document is deleted? (or will it still work since any error is catched and error reply is sent)